.github/workflows/make-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Make full release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publish-to-test-pypi:
+        description: 'Publish to test pypi instead of pypi'
+        required: false
+        default: false
+        type: boolean
+      bump-rule:
+        description: 'Rule for computing next release version'
+        required: false
+        default: 'prerelease'
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+          - prerelease
+      release-version:
+        description: 'Version number to use(instead of computing). Should be of the format x.y.z[rcn]. Do not use hyphens.'
+        required: false
+        default: ''
+        type: string
+      merge-strategy:
+        description: 'Merge strategy and strategy options. Used only in case of merge conflicts'
+        required: false
+        default: ''
+        type: string
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  LANG: en_US.utf-8
+  LC_ALL: en_US.utf-8
+  PYTHON_VERSION: '3.10'
+
+jobs:
+  call-deploy-release:
+    permissions:
+      contents: write  # To push a branch
+      pull-requests: write  # To create a PR from that branch
+
+    uses: eclipse-volttron/github-tooling/.github/workflows/deploy-release.yml@main
+    with:
+      merge-strategy: ${{ inputs.merge-strategy }}
+      release-version: ${{ inputs.release-version }}
+      bump-rule: ${{ inputs.bump-rule }}
+      run-tests-wait: 600
```

Comment:
weren't these going to be not hard coded?

Reply:
have to hard code as github action is changing number type inputs into string when passing the value to reusuable workflow and there doesn't seem to be a typecast function available